src/main/resources/resources/localization/english_en-us.json

```diff
@@ -233,9 +233,9 @@
   "Controls": "Controls",
   "Press the desired": "Press the desired",
   "key sequence": "key sequence",
-  "Confirm setting key bindings.1": "Are you sure you want to reset",
-  "Confirm setting key bindings.2": "all key bindings",
-  "Confirm setting key bindings.3": "to the default keys?",
+  "Confirm resetting key bindings.1": "Are you sure you want to reset",
```

Comment:
Like what @Makkkkus said on Discord, I suggest changing it to “minicraft.popup.reset_keybinds.1” as an example of what I would do.